### DISCUSSION TITLE
hifiberry amp2 hat experiment

### DISCUSSION THREAD

**Discussion by Iorobyy11** [probonopd/MiniDexed#268] [General]
hello!
the project is amazing! a lot of fun! i'm producing a set of new dx7 ambient patches for my songs 
i have only a problem , i use a raspiaudio dac on raspberry pi3 b , sound is very rich and clean but also have a volume that is 
very very low. 
This is a problem when i want to process signal with an external multi efx
...so yesterday i bought a hifiberry amp2 - it seem to use a tas5756m dac - well ...sadly...no sound!   
i've tested and same board work great with another project
it is not supported this hat?
some suggestion? 
i need to increase output volume

**Comment by miotislucifugis** [comment]
Im not familiar with that device but it looks like it is an amplifier and can directly power speakers.   Not sure how you would safely connect that to an effects unit, which presumably takes instrument or line level signals.

**Reply by Iorobyy11** [reply]
I’ve tested the Hi-fiberry amp2 hat with another bare metal synth and processed signal with a zoom multieffects .
I’ve attenuated the signal with a potentiometer and it work good , the signal quality and intensity  is perfect .
The problem is for raspiaudio that have a too low output.
I would to use hifiberry amo with mini Dexed .
This could be perfect for improving the harmonics response .
Fm synthesis could offer a very rich range of harmonics but a good signal intensity is needed.

**Comment by probonopd** [comment]
https://github.com/probonopd/MiniDexed/wiki/Hardware#i2c-dac lists the known working DACs. We don't know whether MiniDexed can work with other DACs. DACs that only use i2s are more likely to work than DACs that use i2s and i2c, which may require an i2c initialization sequence that is different for each DAC model.
